nixops_aws/resources/rds_db_subnet_group.py

```diff
@@ -35,8 +49,8 @@ class RDSDbSubnetGroupState(nixops.resources.ResourceState[RDSDbSubnetGroupDefin
     region = nixops.util.attr_property("region", None)
     description = nixops.util.attr_property("description", None)
     subnet_ids = nixops.util.attr_property("subnet_ids", [], "json")
+    access_key_id = nixops.util.attr_property("accessKeyId", None)
```

Comment:
Why is it required now?

Reply:
It was moved from line 39 to this place so that in the new line 83, we would access it from the defn directly instead of from the config